js-structure_spacerocket/js/launchSequence.js

```diff
@@ -1 +1,20 @@
 // Implement the launch sequence function here and export it as the default export.
+import { NFSAT } from "./payload/satellites.js";
+import { FISHSAT } from "./payload/satellites.js";
+import { fuel } from "./core/fuel.js";
+import { countdown } from "./core/countdown.js";
+import { liftoff } from "./core/liftoff.js";
+import { deployPayload } from "./core/deploy.js";
+
+export function launchFunction() {
```

Comment:
This one is on me. I forgot yesterday to explain the default, but is it in the handout.
For this to work you should add export default function.

Thank you for make me realise this. I just asked Sven if he could explain it to you all.